dev/cnf/oss_dependencies.maven

```diff
@@ -316,6 +316,7 @@ org.eclipse.microprofile.graphql:microprofile-graphql-api:1.0-M3
 org.eclipse.microprofile.health:microprofile-health-api:2.0.1
 org.eclipse.microprofile.health:microprofile-health-api:2.1
 org.eclipse.microprofile.health:microprofile-health-api:2.2-RC2
```

Comment:
Can RC2 be removed?

Reply:
I would have to remove it, once i merge this PR, since I have to delete the binaries in Artifactory as well.